src/malloc/mallocng/malloc.c

```diff
@@ -292,6 +292,7 @@ static struct meta *alloc_group(int sc, size_t req)
 			return 0;
 		}
 		struct meta *g = ctx.active[j];
+		*(char*)(g->mem->storage + get_stride(g) * idx - 4) = 0;
```

Comment:
Add a coment like:
```
// CHEERP: zero-initialize this location, since the code assumes that memory returned by mmap is zero-initialized, but in our case it's not
```

same in the other place below

Reply:
Done.